projects/packages/videopress/src/client/admin/components/video-row/style.module.scss

```diff
@@ -23,9 +23,7 @@
 		background-color: var( --jp-gray-0 );
 
 		.stats:not(.mobile-stats) {
-			opacity: 0;
```

Comment:
I wonder why we were doing that instead of changing the display of the element before. 🤔 

If we want to keep the element in the flow, probably to prevent layout shifts, then we can do `pointer-events: none`.

Reply:
It is weird, indeed. However, I don't have more details or context on when that decision was made.

Thanks for the review.

Reply:
I remember we had some issues with components depending on the Gutenberg version, as Jetpack components are based on them. That _may_ have something to do with it, but I really don't remember.
That said, now it works well :+1: